vividus-to-xray-exporter/src/main/java/org/vividus/xray/databind/ManualTestCaseSerializer.java

```diff
@@ -47,6 +47,8 @@ public void serialize(ManualTestCase manualTest, JsonGenerator generator, Serial
 
         writeObjectWithField(generator, "project", "key", manualTest.getProjectKey());
 
+        writeObjectWithField(generator, "assignee", NAME, manualTest.getAssignee(), true);
```

Comment:
KISS?:
```java
String assignee = manualTest.getAssignee();
if (assignee != null)
{
    writeObjectWithField(generator, "assignee", NAME, manualTest.getAssignee());
}
```

Reply:
fixed, but I thinks sometimes it makes sense to sacrifice KISS in favor of more concise code